adapters/repos/backups/backup_manager_integration_test.go

```diff
@@ -195,3 +235,73 @@ func TestBackupManager_CreateBackup(t *testing.T) {
 		})
 	})
 }
+
+func TestBackupManager_InitModule(t *testing.T) {
+	t.Run("storage-gcs should fail with no bucket set", func(t *testing.T) {
+		ctx, cancel := testCtx()
+		defer cancel()
+
+		require.Nil(t, os.Unsetenv("STORAGE_GCS_BUCKET"))
+
+		rand.Seed(time.Now().UnixNano())
+		rootDir := fmt.Sprintf("./testdata/%d", rand.Intn(10000000))
+		require.Nil(t, os.MkdirAll(rootDir, 0o777))
+
+		compose := docker.New()
+		compose.WithGCS()
+		container, err := compose.Start(ctx)
+		require.Nil(t, err)
+
+		defer func() {
```

Comment:
then this piece of code would be obsolete bc when using `t.TempDir()` it then cleans itself autmatically